R/getRunStatus.R

```diff
@@ -145,7 +145,7 @@ getRunStatus <- function(mydir = dir(), sort = "nf", user = NULL) {
         if (exists("totNoOfIter") && any(grepl("cm_iteration_max", cfg))) if (length(totNoOfIter) > 0 & !cfg[["gms"]][["cm_iteration_max"]] > out[i, "Iter"]) out[i, "Iter"] <- paste0(out[i, "Iter"], "/", sub(";", "", sub("^.*.= ", "", totNoOfIter)))
 
         if (length(suppressWarnings(system(paste0("grep 'Convergence threshold' ", fulllst), intern = TRUE))) > 1) {
-          out[i, "Conv"] <- "converged"
+          out[i, "Conv"] <- if (file.exists(gdx_non_optimal)) "converged (had INFES)" else "converged"
```

Comment:
means that you identify intermediate infes by the existence of a non-optimal gdx, right? @pweigmann do you have any case in mind where this could not work?

Reply:
Yes

Reply:
> means that you identify intermediate infes by the existence of a non-optimal gdx, right? @pweigmann do you have any case in mind where this could not work?

I am not sure, it might also exist if a failed run is restarted in the same folder and then converges without intermediate infes. Unless the restarting deletes the `non_optimal.gdx`?

Reply:
According to [this](https://github.com/remindmodel/remind/blob/develop/start.R#L146), the `non_optimal.gdx` should be moved or renamed to `non_optimal_beforeRestart.gdx`

Reply:
I agree, @whitehacker could could you please implement it in the REMIND start scripst

Reply:
I believe it's already implemented, No? Please have to look to this [line](https://github.com/remindmodel/remind/blob/develop/start.R#L146).

Reply:
yes, sorry I was too focused on the abort.gdx :-)